### ISSUE TITLE
Settings for condensed audio not working for .tsv output

### ISSUE THREAD

**Issue by thymetony** [dxing97/subs2cia#21]
For example, `subs2cia condense -i video.mkv subs.srt -p 500 -t 2500` outputs audio no problem, but `subs2cia srs -i video.mkv subs.srt -p 500 -t 2500` gives the error `TypeError: __init__() missing 2 required positional arguments: 'subtitle_regex_substrfilter' and 'subtitle_regex_substrfilter_nokeep'`. I'm not using the feature which ignores non-dialogue lines and have tried using the command which turns the feature off but with the same result.

Understand flashcard export is an experimental feature but, am I doing something wrong here?

On macOS Catalina.

**Comment by aaron-meyers** [comment]
Seems like 'subs2cia srs' is just broken, I get that error even with just `subs2cia srs -i some-video.mp4 some-subs.ass -d some-dir`

**Comment by aaron-meyers** [comment]
Looking at the commit history, this may have been fixed already and a new version just needs to be pushed to pypi. I'll try installing directly from the repo and see if it works.

**Comment by aaron-meyers** [comment]
Well, the latest version from the repo doesn't give the 'positional arguments' error but it is aborting shortly after starting ffmpeg for my test case (see output below).

It worked with the same files for subs2cia condense (although it took a _very_ long time).

Output:
```
subs2cia:INFO:subs2cia version v0.3.6
subs2cia:INFO:Picked audio stream:    stream 1 (audio, Undetermined, aac) in C:\Data\OneDrive\Watch\Japanese\Movies\Weathering With You\Weathering.With.You.2019.1080p.BluRay.x264.AAC5.1-[YTS.MX].mp4 (video)
subs2cia:INFO:Picked subtitle stream: standalone ass subtitle at tenki-no-ko.ja.ass (subtitle)
subs2cia:INFO:Picked video stream:    stream 0 (video, Undetermined, h264) in C:\Data\OneDrive\Watch\Japanese\Movies\Weathering With You\Weathering.With.You.2019.1080p.BluRay.x264.AAC5.1-[YTS.MX].mp4 (video)
  1%|▊                                                                                           | 16/1745 [00:10<18:06,  1.59it/s]
subs2cia:INFO:Deleting temporary file C:\Data\OneDrive\Watch\Japanese\Movies\Weathering With You\Weathering.With.You.2019.1080p.BluRay.x264.AAC5.1-[YTS.MX].mp4.stream1.audio.und.flac
```

**Comment by dxing97** [comment]
Should be fixed in latest master and v0.4.0 on pypi.
